examples/angular2/tsconfig.json

```diff
@@ -1,15 +1,13 @@
 {
   "compilerOptions": {
-    "target": "ES5",
+    "target": "es5",
     "module": "commonjs",
     "moduleResolution": "node",
     "sourceMap": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
-    "noImplicitAny": true
-  },
-  "files": [
-    "app/bootstrap.ts"
-  ]
+    "noImplicitAny": true,
+    "suppressImplicitAnyIndexErrors": true
+  }
```

Comment:
Can we specify a different --outDir so that the transpiled JS files don't mix with the TS files? I think it will be easier for everyone to inspect the source code that way. What do you guys think?